packages/frontend/src/components/LoginForm.tsx

```diff
@@ -32,24 +31,50 @@ import SettingsSwitch from './Settings/SettingsSwitch'
 
 const log = getLogger('renderer/loginForm')
 
+export type Credentials = T.EnteredLoginParam & ProxySettings
+
+type ProxySettings = {
+  proxyEnabled: boolean
+  proxyUrl: string
```

Comment:
```suggestion
  proxyEnabled: boolean
  /** Note that this could be `null` */
  proxyUrl: string
```

See https://github.com/deltachat/deltachat-desktop/pull/4945#discussion_r2039867865

Reply:
I changed the type to string | null